bin/ember-template-lint.js

```diff
@@ -169,6 +169,11 @@ function parseArgv(_argv) {
         describe: 'Prevent inline configuration comments from changing config or rules',
         boolean: true,
       },
+      'max-warnings': {
+        describe: 'Number of warnings to trigger nonzero exit code',
+        type: 'number',
+        default: -1,
```

Comment:
Let's remove the default value. If we do, then we can do a simpler comparison (`options.maxWarnings && results.warningCount > options.maxWarnings`) without having to check for `-1`.

Also, to a user/human a value of `-1` is somewhat "odd" (like when they read the help output). If I read it, I'd assume that we actually wanted to error for any warnings! (which is definitely not your intent here).